src/librustdoc/lint.rs

```diff
@@ -158,6 +161,22 @@ declare_rustdoc_lint! {
     "detects URLs that could be written using only angle brackets"
 }
 
+declare_rustdoc_lint! {
+    /// The `doc_include` lint detects when `#[doc(include = ...)]` is used.
+    /// This feature is scheduled for removal and will give a hard error in a future release.
+    ///
+    /// This is a `rustdoc` only lint, see the documentation in the [rustdoc book].
+    ///
+    /// [rustdoc book]: ../../../rustdoc/lints.html#non_autolinks
```

Comment:
This link is incorrect:

```suggestion
    /// [rustdoc book]: ../../../rustdoc/lints.html#doc_include
```